xml/System.Net.Quic/QuicConnection.xml

```diff
@@ -205,11 +228,11 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="type">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="type">The type of the stream, either unidirectional or bidirectional.</param>
+        <param name="cancellationToken">A cancellation token that can be used to cancel the asynchronous operation.</param>
+        <summary>Creates an outbound unidirectional or bidirectional <see cref="T:System.Net.Quic.QuicStream" />.</summary>
+        <returns>An asynchronous task that completes with the opened <see cref="T:System.Net.Quic.QuicStream" />.</returns>
+        <remarks>If the connection doesn't have any available stream capacity, that is, the peer limits the concurrent stream count, the operation pends until the stream can be opened (that is, when the other stream is closed or the peer increases the limit).</remarks>
```

Comment:
I would not mention closing any streams, as other QUIC implementation don't have to immediately respond by increasing the stream limit when a peer closes a locally initiated stream.

```suggestion
        <remarks>If the connection doesn't have any available stream capacity, that is, the peer limits the concurrent stream count, the operation pends until the peer increases the stream limit.</remarks>
```

Reply:
Then we should also not mention increasing the stream limit either.